src/nlp.jl

```diff
@@ -308,115 +308,114 @@ function MOI.initialize!(d::NLPEvaluator, requested_features::Vector{Symbol})
 
     d.parameter_values = nldata.nlparamvalues
 
-    tic()
-
-    d.want_hess = (:Hess in requested_features)
-    want_hess_storage = (:HessVec in requested_features) || d.want_hess
-    coloring_storage = Derivatives.Coloring.IndexedSet(num_variables_)
-
-    d.has_nlobj = isa(nldata.nlobj, NonlinearExprData)
-    max_expr_length = 0
-    main_expressions = Array{Vector{NodeData}}(undef,0)
-    subexpr = Array{Vector{NodeData}}(undef,0)
-    for nlexpr in nldata.nlexpr
-        push!(subexpr, nlexpr.nd)
-    end
-    if d.has_nlobj
-        push!(main_expressions,nldata.nlobj.nd)
-    end
-    for nlconstr in nldata.nlconstr
-        push!(main_expressions,nlconstr.terms.nd)
-    end
-    d.subexpression_order, individual_order = order_subexpressions(main_expressions,subexpr)
-
-    d.subexpression_linearity = Array{Linearity}(undef,length(nldata.nlexpr))
-    subexpression_variables = Array{Vector{Int}}(undef,length(nldata.nlexpr))
-    subexpression_edgelist = Array{Set{Tuple{Int,Int}}}(undef,length(nldata.nlexpr))
-    d.subexpressions = Array{SubexpressionStorage}(undef,length(nldata.nlexpr))
-    d.subexpression_forward_values = Array{Float64}(undef,length(d.subexpressions))
-    d.subexpression_reverse_values = Array{Float64}(undef,length(d.subexpressions))
-
-    empty_edgelist = Set{Tuple{Int,Int}}()
-    for k in d.subexpression_order # only load expressions which actually are used
-        d.subexpression_forward_values[k] = NaN
-        d.subexpressions[k] = SubexpressionStorage(nldata.nlexpr[k].nd, nldata.nlexpr[k].const_values, num_variables_, d.subexpression_linearity, moi_index_to_consecutive_index)
-        subex = d.subexpressions[k]
-        d.subexpression_linearity[k] = subex.linearity
-        @assert subex.linearity != CONSTANT
-        if d.want_hess
-            empty!(coloring_storage)
-            compute_gradient_sparsity!(coloring_storage,subex.nd)
-            # union with all dependent expressions
-            for idx in list_subexpressions(subex.nd)
-                union!(coloring_storage, subexpression_variables[idx])
+    tprep = @elapsed begin
```

Comment:
tprep isn't actually used anywhere, so I'd prefer to delete it and the commented-out `println` instead of introducing the `@elapsed` block.